config/crd/bases/infrastructure.cluster.x-k8s.io_nutanixclusters.yaml

```diff
@@ -27,6 +27,9 @@ spec:
       jsonPath: .status.ready
       name: Ready
       type: string
+    deprecated: true
+    deprecationWarning: v1alpha4 is deprecated and set as unserved and  will be removed
```

Comment:
nit: extra space between "and" and "will" in the stmt

Reply:
good catch. Updated to remove the space and updated the comment to be more descriptive.